Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Facebook Service/FacebookPage.xaml.cs

```diff
@@ -73,7 +73,7 @@ public FacebookPage()
 
             if (QueryType.SelectedIndex == 3)
             {
-                PhotoGridView.ItemsSource = await FacebookService.Instance.GetUserAlbumsAsync();
+                PhotoGridView.ItemsSource = await FacebookService.Instance.GetUserAlbumsAsync(20, FacebookAlbum.Fields);
```

Comment:
to maintain backward compat shouldn't we have an overload of GetUserAlbumsAsync that takes no params ?